SU-SWS/ace-sdssgryphon
# SDSS-538: changing caption design

# READY FOR REVIEW

# Summary
- changing design of captions

# Review By (Date)
- 4/10/2023

# Urgency
- normal

# Steps to Test

1. Pull in this change.
2. Update and clear caches.
3. Verify the font matches the design on these two pages `/qa-typography-styles` and `news/qa-news-article-plage`

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
[- SDSS-538](https://stanfordits.atlassian.net/browse/SDSS-538)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
